src/parse.c

```diff
@@ -4440,7 +4440,7 @@ void Parser::checkCstyleTypeSyntax(Loc loc, Type *t, int alt, Identifier *ident)
 
 Statement *Parser::parseStatement(int flags, const utf8_t** endPtr, Loc *pEndloc)
 {
-    Statement *s;
+    Statement *s = NULL;
```

Comment:
> The other cases all also emit errors and could be changed to return/goto Lerror as far as I can see.

This is getting out of my comfort zone. I'm thinking I'll fix the bug that the warning found, and you can implement your fix when you have time.